### PR TITLE
Replace native Alerts with a Toast

### PR DESCRIPTION
**Pull request by filiptronicek**
Awesone toasts. Let's toast to the toasts!

- [ ] Dark mode would be cool

### CODE REVIEW

**Comment by bigint** [comment]
Love to see screenshots 🤗

**Comment by filiptronicek** [comment]
Sure thing, @bigint!
image
image

File: pages/FilePage.js [hunk 1]
```diff
@@ -45,7 +45,13 @@ export function FilePage() {
         await ImagePicker.requestMediaLibraryPermissionsAsync();
 
       if (permissionResult.granted === false) {
-        Alert.alert("Permission to access camera roll is required!");
+        Toast.show({
+          type: "error",
+          text1: "Error!",
+          text2: `Permission to access the camera roll is required!`,
```
Comment (bigint) [inline pages/FilePage.js:51]: @filiptronicek Do you think backtick is needed here? If you see `text1` its in dquote 🤔

File: pages/FilePage.js [hunk 2]
```diff
@@ -69,7 +75,13 @@ export function FilePage() {
       const permissionResult = await ImagePicker.getCameraPermissionsAsync();
 
       if (permissionResult.granted === false) {
-        Alert.alert("Permission to access the camera is required!");
+        Toast.show({
+          type: "error",
+          text1: "Error!",
+          text2: `Permission to access the camera is required!`,
```
Comment (bigint) [inline pages/FilePage.js:81]: Here too!

File: pages/FilePage.js [hunk 3]
```diff
@@ -135,7 +151,13 @@ export function FilePage() {
               return res.json();
             } else {
               Haptics.notificationAsync(Haptics.NotificationFeedbackType.Error);
-              Alert.alert("Error!", `Got the error ${res.status}.`);
+              Toast.show({
+                type: "error",
+                text1: "Error!",
+                text2: `Got the error ${res.status}.`,
```
Comment (bigint) [inline pages/FilePage.js:157]: Suitable for this scenario :D

File: pages/HomeScreen.js [hunk 4]
```diff
@@ -136,18 +136,27 @@ export function HomeScreen({ navigation }) {
...
           } else {
             if (response.status === 429) {
-              Alert.alert(
-                "Slow down!",
-                "We are getting too many requests from you."
-              );
+              Toast.show({
+                type: "error",
+                text1: "Slow down!",
+                text2: "We are getting too many requests from you.",
```
Comment (bigint) [inline pages/HomeScreen.js:142]: Purrrfect!

**Review by bigint — CHANGES_REQUESTED** [review]

**Comment by bigint** [comment]
Others LGTM 🚀

**Comment by filiptronicek** [comment]
Dark mode maybe later, aligator

**Pull request merged**